mentat/terminal/output.py

```diff
@@ -29,13 +31,22 @@ def print_stream_message(message: StreamMessage):
     end = "\n"
     color = None
     flush = False
+    session_context = SESSION_CONTEXT.get()
```

Comment:
I think we want to avoid using `SESSION_CONTEXT` on the clients. The terminal holds the config which has the theme. Can you get it from there instead please?

Reply:
removed

Reply:
sorry for the wrong one.